data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/reconciler/impl/ResourcesReconcilerMessageHandler.java

```diff
@@ -35,44 +35,63 @@
 
   public final static String ADDRESS = "resourcesreconciler.core";
 
-  private final Vertx vertx;
   private final ResourcesReconciler resourcesReconciler;
-  private final AtomicBoolean scheduled;
   private final AtomicBoolean reconciling;
   private final AtomicReference<DataPlaneContract.Contract> last;
 
-  public ResourcesReconcilerMessageHandler(final Vertx vertx, final ResourcesReconciler resourcesReconciler) {
-    this.vertx = vertx;
+  public ResourcesReconcilerMessageHandler(final ResourcesReconciler resourcesReconciler) {
     this.resourcesReconciler = resourcesReconciler;
     reconciling = new AtomicBoolean();
-    scheduled = new AtomicBoolean();
     last = new AtomicReference<>();
   }
 
   @Override
   public void handle(Message<Object> event) {
     DataPlaneContract.Contract contract = (DataPlaneContract.Contract) event.body();
     last.set(contract);
```

Comment:
Can you move this in the function below? It helps the reasoning

Reply:
Sure!